### PR TITLE
completed User database task by using sqlalchemy

### PR DESCRIPTION
**Pull request by srikar-1997**
please review the task of the user database with orm

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
 from flask_session import Session
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
+import logging
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import exc
+
 
 app = Flask(__name__)
+os.environ["DATABASE_URL"] = "postgres://vdsjxpbuoklrgr:1669a5d62c3c6e5aaf3c580c4245a3703c760912beb63a870d62f93c69e6b45f@ec2-54-88-130-244.compute-1.amazonaws.com:5432/d8ijt27ijeed12"
```
Comment (rishikaharithaMSIT) [inline project1/application.py]: Donot store url in code ... anyone who can see the code will get access to your database.

File: project1/application.py [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
+# app.config["SESSION_PERMANENT"] = False
+# app.config["SESSION_TYPE"] = "filesystem"
+# Session(app)
 
-# Set up database
-engine = create_engine(os.getenv("DATABASE_URL"))
-db = scoped_session(sessionmaker(bind=engine))
+# # Set up database
+# engine = create_engine(os.getenv("DATABASE_URL"))
+# db = scoped_session(sessionmaker(bind=engine))
```
Comment (rishikaharithaMSIT) [inline project1/application.py]: remove unused lines

File: project1/templates/layout.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        <title>{% block title %}{% endblock %}</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    </head>
+    <body>
+        <div class="container">
+            {% block body %}
+            {% endblock %}
+        </div>
+    </body>
+</html>
```
Comment (rishikaharithaMSIT) [inline project1/templates/layout.html:13]: Good work! You used layouts

**Review by rishikaharithaMSIT — CHANGES_REQUESTED** [review]

File: project1/application.py (hunk 1 above)
Comment (srikar-1997) [reply project1/application.py]: ok, removed the url.

File: project1/templates/layout.html (hunk 3 above)
Comment (srikar-1997) [reply project1/templates/layout.html:13]: Thank You

**Review by rishikaharithaMSIT — APPROVED** [review]

**Pull request merged**